frontend/src/Editor/Inspector/Utils.js

```diff
@@ -28,10 +29,24 @@ export function renderElement(
   components = {},
   darkMode = false
 ) {
-  const componentDefinition = component.component.definition;
+  const componentConfig = component.component;
+  const componentDefinition = componentConfig.definition;
   const paramTypeDefinition = componentDefinition[paramType] || {};
   const definition = paramTypeDefinition[param] || {};
 
+  if (componentConfig.component == 'DropDown') {
+    const paramTypeConfig = componentMeta[paramType] || {};
+    const paramConfig = paramTypeConfig[param] || {};
+    const { conditionallyRender = null } = paramConfig;
+
+    if (conditionallyRender) {
+      const { key, value } = conditionallyRender;
+      if (paramTypeDefinition?.[key]) {
```

Comment:
```suggestion
      if (paramTypeDefinition?.[key] ?? value) {
```

Without value, it is rendering both `schema` and other values for older widgets